### PR TITLE
Added 'role-addition' tox environment to enable external contributions

### PR DESCRIPTION
**Pull request by rebtoor**
Signed-off-by: Roberto Alfieri <ralfieri@redhat.com>

* imported zuul.d, _skeleton_role_ and playbooks/roles/plugins needed by zuul/molecule

### CODE REVIEW

File: zuul.d/base.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (raukadah) [inline zuul.d/base.yaml:1]: We can drop this file as we will be running molecule jobs via prow.

File: zuul.d/layout.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (raukadah) [inline zuul.d/layout.yaml:1]: Same here.

File: zuul.d/molecule.yaml [hunk 3]
```diff
@@ -0,0 +1,310 @@
+# Managed via ./role-addition.yml, do not edit manually without testing that
```
Comment (raukadah) [inline zuul.d/molecule.yaml:1]: same here.

**Review by raukadah — CHANGES_REQUESTED** [review]
Thank you for adding this. I will pull this change locally and add more comments. For till now, we can drop zuul specific code from here. Since molecule jobs will run via prow.

**Comment by raukadah** [comment]
recheck

File: zuul.d/base.yaml (hunk 1 above)
Comment (rebtoor) [reply zuul.d/base.yaml:1]: ack

File: zuul.d/layout.yaml (hunk 2 above)
Comment (rebtoor) [reply zuul.d/layout.yaml:1]: ack

File: zuul.d/molecule.yaml (hunk 3 above)
Comment (rebtoor) [reply zuul.d/molecule.yaml]: ack

**Comment by jpodivin** [comment]
I'm seeing reconciler error in maria logs [0]

`1.6762775400825953e+09	ERROR	Reconciler error	{"controller": "mariadb", "controllerGroup": "mariadb.openstack.org", "controllerKind": "MariaDB", "MariaDB": {"name":"openstack","namespace":"openstack"}, "namespace": "openstack", "name": "openstack", "reconcileID": "592d9eca-c33e-493c-a356-ad7538abf5fd", "error": "persistentvolumeclaims \"mariadb-openstack\" already exists"}`

[0]https://logserver.rdoproject.org/35/35/260e36e8d73eda4a8ca9aa0961c8a1663cdb5e9b/github-check/edpm-ansible-github-rdo-integration-centos-8-crc-singlenode-centos-9-external-compute/32f747c/logs/controller/pod/mariadb-operator-controller-manager-5b7fd764c9-zpm5b-logs.txt

**Comment by raukadah** [comment]
recheck

**Pull request closed without merging**